.rubocop.yml

```diff
@@ -3,18 +3,10 @@ AllCops:
     - vendor/**/*
     - Guardfile
 
-AlignParameters:
```

Comment:
Ah ok, i will correct the cop titles then (with the type/cop syntax) and put them back in